client/src/Main.elm

```diff
@@ -9,12 +9,12 @@ import View
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
-    Time.every (Time.second * 2) UpdateMode
+    Time.every 2000 UpdateMode
 
 
-main : Program Never Model Msg
+main : Program () Model Msg
```

Comment:
The second argument is for `Flags`, which isn't being used. We could do this in the future maybe, but it might not be worth it. We could set the `mode` with a flag, but we'll still need to query the DB for when it changes, unless we want to tell people to refresh their browsers.